ui/app/routes/vault/cluster.js

```diff
@@ -116,5 +105,8 @@ export default Route.extend(ModelBoundaryRoute, ClusterRoute, {
       }
       return true;
     },
+    deactivate() {
+      this.poll.cancelAll();
```

Comment:
Oh nice - I'd missed that part of the docs - also makes me think that we probably want `keepLatest` for the concurrency model instead of `drop`. Thanks!